### ISSUE TITLE
Remove REST acceptance tests in favor of always live monitoring tool

### ISSUE THREAD

**Issue by apeksharma** [hashgraph/hedera-mirror-node#627]
Monitor covers all the tests cases in acceptance tests.
Additionally,
- monitor is always running and checking health of testnet and mainnet every minute.
- Once #424 is done, monitor can be released as an image and run in a K8 cluster

Acceptance tests are no longer needed.
Kill it with 🔥

This also means that running REST acceptance test won't be needed as part of release process. Which means that in platform-only release, there wouldn't be need of any product eng (who were needed till now just to run the tests). 
This will save process and eng time.
